submissions/TaniaBondarenko/dom_task/my_javascript.js

```diff
@@ -0,0 +1,40 @@
+let i = 0; let n = 12;
+let content = document.getElementsByClassName("content");
```

Comment:
It's better to use const here because you didn't any of the reassign of this variable

Reply:
fixed